editor.planx.uk/src/@planx/components/FindProperty/Public/Map.tsx

```diff
@@ -128,6 +131,7 @@ export default function PlotNewAddress(props: PlotNewAddressProps): FCReturn {
           showScale
           showNorthArrow
           osProxyEndpoint={`${process.env.REACT_APP_API_URL}/proxy/ordnance-survey`}
+          clipGeojsonData={JSON.stringify(boundaryBBox)}
```

Comment:
This is the remaining place where the team's boundary is used - on `PlotNewAddress`